src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const number1 = Big(numberOne);
+  const number2 = Big(numberTwo);
+  let result = Big(0);
+  switch (operation) {
+    case '+':
+      result = number1 + number2;
+      break;
+    case '-':
+      result = number1 - number2;
+      break;
+    case 'x':
+      result = number1 * number2;
+      break;
+    case '÷':
+      result = number1 / number2;
+      break;
+    case '%':
+      result = number1 + 0.01;
```

Comment:
* [ ] adding 0.01 to number1 won't convert the number to decimal but by multiplying, let us address this issue .

Reply:
You are absolutely right, it was a mistake